msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -32,6 +32,7 @@
 using Microsoft.Identity.Client.Internal;
 using Microsoft.Identity.Client.Internal.Requests;
 using System.Linq;
+using System.Reflection;
```

Comment:
>ing System.Reflection; [](start = 2, length = 22)

why is this needed?

Reply:
We moved AssemblyInformationalVersionAttribute in here (with the GetAssemblyInformationalVersion method) and that's in the system.reflection namespace.